assets/src/edit-story/components/library/libraryLayout.js

```diff
@@ -6,12 +6,12 @@ import styled from 'styled-components';
 /**
  * Internal dependencies
  */
-import { HEADER_HEIGHT } from '../../constants';
+import { ADMIN_TOOLBAR_HEIGHT, HEADER_HEIGHT } from '../../constants';
 import LibraryContent from './libraryContent';
 import LibraryTabs from './libraryTabs';
 
 const Layout = styled.div`
-	height: 100%;
+	height: calc(100vh - ${ ADMIN_TOOLBAR_HEIGHT }px);
```

Comment:
What if the admin bar is disabled? Will this then break?

Reply:
There are several ways to disable the admin. [See](https://developer.wordpress.org/reference/functions/show_admin_bar/#comment-1300)

Reply:
The more you know. Fair enough.